CHANGELOG.md

```diff
@@ -2,10 +2,17 @@
 
 ## 0.9.28
 
+### New features
+
+* [#3862](https://github.com/buefy/buefy/issues/3862) Expose `isActiveDetailRow` method in the default scoped slot props of table column (thanks @OrestHk)
+* [#3863](https://github.com/buefy/buefy/issues/3863) Add `active` event emitted when dropdown turns active/inactive (thanks @daniil4udo)
+
 ### Fixes
 
```

Comment:
@kikuomax are we waiting for results for #3992 to add #3991 to the list of fixes?

Reply:
@wesdevpro No, I do not expect we can conclude #3992 before releasing 0.9.28.

Reply:
@kikuomax Makes sense to me 👍🏼Just wanted to confirm with you